Makefile

```diff
@@ -2,7 +2,7 @@ TEST?=./...
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
 
 #default: fmt test testrace vet
-default: fmtcheck vet build copyright
+default: fmtcheck vet build
```

Comment:
Why did you remove the check, considering that you recently enhanced the copyright check to be smarter and only check recently changed files?

Reply:
The check [failed unexpectedly](https://travis-ci.org/vmware/go-vcloud-director/builds/475189768) on a file that was not modified recently. I need to understand why. 
I think that my algorithm works on the assumption of a centralized source of truth, but fails on a truly distributed repository.